<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPerepis</class>
 <widget class="QDialog" name="addPerepis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>567</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">max-width:60;</string>
       </property>
       <property name="text">
        <string>Кабинет</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">max-width:50;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">max-width:37;</string>
       </property>
       <property name="text">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">max-width:350;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">max-width:21;</string>
       </property>
       <property name="text">
        <string>УЗ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">max-width:180;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>170</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="6" colspan="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Блокхост</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>eToken</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Инвентарный номер</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="4">
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="4">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Розетка тел.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="4">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="viewPK">
       <property name="styleSheet">
        <string notr="true">max-width:30</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img/img.qrc">
         <normaloff>:/24-03-2015-491.png</normaloff>:/24-03-2015-491.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="4">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="4">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Коментрий</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Серийный номер</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Операционная система</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="5">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="4" colspan="5">
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="3">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Имя ПК</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="3">
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="viborMon">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Серийный номер</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Монитор   </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Компьютер</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="3">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Инвентарный номер</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="8">
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
     <item row="11" column="2" colspan="3">
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="viborPC">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="4">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Розетка ПК</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5" colspan="2">
      <widget class="QPushButton" name="viewMon">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">max-width:30</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img/img.qrc">
         <normaloff>:/24-03-2015-491.png</normaloff>:/24-03-2015-491.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="8">
      <widget class="QLabel" name="label_MON_st">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="8">
      <widget class="QLabel" name="label_BH_st">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>351</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="exit">
       <property name="text">
        <string>    Отмена   </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
</ui>
